pkg/build/controller/pipeline_controller.go

```diff
@@ -0,0 +1,60 @@
+package controller
+
+import (
+	"fmt"
+
+	"github.com/golang/glog"
+
+	buildapi "github.com/openshift/origin/pkg/build/api"
+	buildclient "github.com/openshift/origin/pkg/build/client"
+	kclient "k8s.io/kubernetes/pkg/client/unversioned"
+)
+
+// JenkinsPipelineServiceName is the name of the Jenkins service this pipeline
+// deploys.
+const JenkinsPipelineServiceName = "ci"
```

Comment:
since we're being specific, we should include jenkins in this name.  jenkins-ci (if that's a legal name)?